dds_web/web/user.py

```diff
@@ -148,6 +152,102 @@ def register():
     return flask.render_template("user/register.html", form=form)
 
 
+@auth_blueprint.route("/activate_totp/<token>", methods=["GET", "POST"])
+@limiter.limit(
+    dds_web.utils.rate_limit_from_config,
+    methods=["GET", "POST"],
```

Comment:
Is this part needed?

Reply:
Only the `methods=["GET", "POST"]`  - it's defined in the route

Reply:
Also not a big thing though, just a question

Reply:
Wouldn't it be for the limiter in this case? The  `confirm_2fa()` method is written the same way.

Reply:
Yeah my question regards both of them. All the other endpoints aren't written like that so I'm assuming it's not needed.